### ISSUE TITLE
Open new instance of tilda in new tab

### ISSUE THREAD

**Issue by saaa-hsjgrobler** [lanoxx/tilda#145]
Hi, is it possible to make it so that whenever a new instance of tilda is added, it instead adds a new tab to the existing tilda instance (if there is one)?

I can't seem to find anything in the preferences and there is one similar, badly worded, request as well as an older request that isn't very helpful.

**Comment by pik** [comment]
You can right-click or use the new-tab hotkey open a new-tab or do you mean doing so via. terminal command?

**Comment by saaa-hsjgrobler** [comment]
Yes, or in my case when I click "Open new terminal here" from within my
file explorer (nemo), I would like it to open in a new tab instead of a new
window.

Kind Regards

HENDRIK GROBLER
IT Manager
tel: 011 470 2930cell: 082 941 3701email: it‑support@accountingacademy.co.za
<it-support@accountingacademy.co.za>

www.accountingacademy.co.za

**Comment by lanoxx** [comment]
This required dbus support in tilda first.

**Comment by canonical-is** [comment]
+1

**Comment by alejdg** [comment]
+1

**Comment by renyhp** [comment]
+1. This seems an old issue but quite important, nothing yet?

**Comment by TheStrangeKat** [comment]
Lack of a solution also seems to break "xdg-open" functionality (or at the very least makes it throw a fit when running it). Just thought it worth mentioning.

**Comment by cydouzo** [comment]
+1, This is a huge issue, for what would be otherwise the indisputable best drop-down terminal around in my opinion.

**Comment by lanoxx** [comment]
I have created branch [wip-dbus](https://github.com/lanoxx/tilda/tree/wip-dbus) and started to experiment with dbus. It is not yet complete, but it already works for pulling the tilda window up and down.

After compiling you can pull the window with 

    dbus-send --session --print-reply --dest=com.github.lanoxx.tilda.Actions1 /com/github/lanoxx/tilda/Actions1 com.github.lanoxx.tilda.Actions1.Toggle

Note that the 1 in `Actions1` refers to the tilda instance, so if you run two tilda instances you can control them separately with `Actions1` and `Actions2`.

**Comment by chexmo** [comment]
This would be a huge improvement to tilda. This issue spots the only fact i really don't like about tilda.
